composer.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "test": "vendor/bin/phpunit",
     "lint": "vendor/bin/phpcs",
-    "format": "vendor/bin/phpcbf"
+    "format": "vendor/bin/phpcbf",
+    "check": "./vendor/bin/phpstan analyse --level max src tests"
```

Comment:
By running `composer check` I got a few errors that needs to be addressed:

![CleanShot 2022-10-22 at 18 19 27](https://user-images.githubusercontent.com/19310830/197350292-9980e136-2bc9-48dc-b6f7-9aedab8c8b64.png)

Reply:
I have fixed errors till level 8. Can you please look into it now?